packages/sdk/src/methods/token.ts

```diff
@@ -44,17 +45,10 @@ export async function tokenInfo(handle: string, params?: ProviderParams) {
     return undefined;
   }
 
-  const { value: tokenName } = await registry.functions
-    .name({ bits: ZeroBytes32 })
-    .get();
-  const { value: tokenSymbol } = await registry.functions
-    .symbol({ bits: ZeroBytes32 })
-    .get();
+  registry.functions.metadata({ bits: assetId }, 'image:png');
```

Comment:
this method was called but the return value was not captured, it should return the token image.